Docs/source/Software/BSP.rst

```diff
@@ -135,3 +135,19 @@ Usage
 Additional Considerations
     Increasing the clock speed does take more power. Although this is minor considering how much the
     entire BPS takes, it is worth noting.
+
+SPI BSP: Clark Poon, Sijin Woo, and Sugam Arora
+===============================================
+
+Purpose
+    SPI is used for communication between the uC and the minion LTC6811s in the system.
+
+Usage
+    Our car uses the ``SPI1`` and ``SPI3`` busses. The ``SPI1`` bus is for communication with the LTC6811s
+    and the ``SPI3`` bus is for communication with the AS8510 current sensor. When calling the init function, 
+    you must specify which port you intend to use (via the ``spi_port_t`` struct). The user can read/write on whichever bus they 
```

Comment:
If there are #define for BareMetal and RTOS, I would explain why they are there and how to use them for each version